pgnotify/__init__.py

```diff
@@ -1,9 +1,23 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
+
 from logx import log
 
-log.set_null_handler()
-from .notify import (  # noqa
+from .notify import (
     await_pg_notifications,
-    get_dbapi_connection,
+    get_dbapi_connection,  # noqa
     start_listening,
 )
+
+log.set_null_handler()
+
+
+def notify(connection, channel, payload):
+    """Send a PostgreSQL notify with a payload
+
+    :param connection: dburi, sqlengine or dbapiconnection
+    :param channel: channel that the notify is sent to
+    :param payload: payload to be sent together with notify
+    """
+    connection = get_dbapi_connection(connection)
+    cursor = connection.cursor()
+    cursor.execute("NOTIFY %s, '%s';" % (channel, payload))
```

Comment:
This is vulnerable to SQL-injection (correct me if I'm wrong 😉). Is there a way to fix this? If not, it should be explicitly stated in the documentation that untrusted payloads / channels aren't supported.

Reply:
Also it seems to be better to use the `pg_notify` function in Postgres for sending notifications to non-constant channel names and payloads?

Refer to the `Notes` section in the doc: https://www.postgresql.org/docs/9.0/sql-notify.html